docker-wrappers/RWR/README.md

```diff
@@ -0,0 +1,29 @@
+# RWwR Docker image
+
+A Docker image for the random-walk-with-start algorithm that is available on [DockerHub](https://hub.docker.com/repository/docker/erikliu24/rwwr).
+
+To create the Docker image run:
+```
+docker build -t eriliu24/RandomWalk -f Dockerfile .
+```
+from this directory.
+
+To inspect the installed Python packages:
+```
+winpty docker run erikliu24/rwwr pip list
+```
+The `winpty` prefix is only needed on Windows.
+
+## Testing
+Test code is located in `test/RandomWalk`.
+The `input` subdirectory contains test files `source_nodes.txt`, `target_nodes.txt` and `edges.txt`.
+The Docker wrapper can be tested with `pytest`.
```

Comment:
you could use pytest -k test_rwr.py rather than just pytest (but I think either or can be written)